ndcube/ndcube.py

```diff
@@ -199,7 +220,14 @@ def crop_by_values(self,
                        wcs: Optional[Union[BaseHighLevelWCS, ExtraCoordsABC]] = None
                        ) -> "NDCubeABC":
         """
-        Crop to the smallest cube in pixel space containing the world coordinate points.
+        Crop using real world coordinates.
+        This method crops the NDCube to the smallest bounding box in pixel space that
+        contains all the provided world coordinate points.
+
+        This function takes points as iterables of low-level coordinate objects, i.e. `~astropy.units.Quantity` objects.
+        This differs from the `.crop()` method which takes high-level coordinate objects that contain
+        all the relevant coordinate information such as coordinate frame etc.
```

Comment:
Same comment as above regarding being explicit when linking to crop's documentation.  (In making this suggestion I've also minimally tidied up the text and line lengths.
```suggestion
        This function takes points as iterables of low-level coordinate objects,
        i.e. `~astropy.units.Quantity` objects.  This differs from `~ndcube.NDCube.crop()` 
        which takes high-level coordinate objects requiring all the relevant coordinate
        information such as coordinate frame etc.  Hence this method's API is more basic
        but less explicit.
```